Shared/Models/Notification.cs

```diff
@@ -3,9 +3,11 @@
     public class Notification
     {
         public int IdUser { get; set; }
-        public int IdQuestion { get; set; }
+        public bool IsSeen { get; set; }
+        public int Type { get; set; }
+        public int? IdEntity { get; set; }
```

Comment:
Please add comment saying that IdEntity value should be either IdAnswer or IdQuestion or NULL if notification type refers to premium notifications